internal/pkg/daemon/bpfrecorder/bpfrecorder_apparmor.go

```diff
@@ -203,6 +203,27 @@ func (b *AppArmorRecorder) handleCapabilityEvent(capEvent *bpfEvent) {
 	b.recordedCapabilities[mid] = append(b.recordedCapabilities[mid], requestedCap)
 }
 
+// Delete all data recorded for a particular mount namespace.
+//
+// The recorder triggers this after container initialization to make sure that
+// permissions needed for setup are not included in the final profile.
+func (b *AppArmorRecorder) clearMntns(event *bpfEvent) {
+	// Everything we captured so far is container initialization.
+	// Let's get rid of it.
+	b.lockRecordedFiles.Lock()
+	defer b.lockRecordedFiles.Unlock()
+	b.lockRecordedCapabilities.Lock()
+	defer b.lockRecordedCapabilities.Unlock()
+	b.lockRecordedSocketsUse.Lock()
+	defer b.lockRecordedSocketsUse.Unlock()
+
+	mntns := mntnsID(event.Mntns)
+	log.Printf("Clearing mntns: %d\n", mntns)
```

Comment:
Does the syscalls maps doesn't need any clean-up? Is it already cleaned in the eBPF code?

Reply:
Already cleaned in the eBPF code.